### DISCUSSION TITLE
Show Loader

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#251] [3. Integrations]
Hi @vovayatsyuk 

How we can show the loader while Ajax requests?

Loader is not showing by using this ``` showLoader: true ``` 

```
 $.ajax({
                type: "GET",
                showLoader: true,
                success: function (response) {
                  
                },
                error: function () {
                    
                }
            });
```

**Comment by vovayatsyuk** [comment]
Use spinner on the related element instead of full page loader animation: https://breezefront.com/docs/spinner
